automation/update.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+remote_repo_status() {
+    UPSTREAM="origin/master"
+    LOCAL=$(git rev-parse @)
+    REMOTE=$(git rev-parse "$UPSTREAM")
+    BASE=$(git merge-base @ "$UPSTREAM")
+
+    if [ $LOCAL = $REMOTE ]; then
+        # "Up-to-date"
+        return 0
+    elif [ $LOCAL = $BASE ]; then
+        # "Need to pull"
+        return 1
+    elif [ $REMOTE = $BASE ]; then
```

Comment:
Do we have a use case for having local changes that are comitted?

Reply:
I sometimes edit code directly on Raspberry Pi.